Library/Formula/open-mpi.rb

```diff
@@ -50,9 +55,11 @@ def install
     # (Fortran header) in `lib` that need to be moved to `include`.
     include.install Dir["#{lib}/*.mod"]
 
-    # Move vtsetup.jar from bin to libexec.
-    libexec.install bin/"vtsetup.jar"
-    inreplace bin/"vtsetup", "$bindir/vtsetup.jar", "$prefix/libexec/vtsetup.jar"
+    if build.stable?
```

Comment:
`vtsetup.jar` doesn't exist in HEAD, or at least isn't in the same place, so it causes the build to fail.